src/login/logind-session-dbus.c

```diff
@@ -875,7 +875,7 @@ static const sd_bus_vtable session_vtable[] = {
         SD_BUS_PROPERTY("VTNr", "u", NULL, offsetof(Session, vtnr), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("Seat", "(so)", property_get_seat, 0, SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("TTY", "s", NULL, offsetof(Session, tty), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
`TTY` likely needs the same treatment.

Reply:
-> https://github.com/systemd/systemd/pull/30053